openmetadata-spec/src/main/resources/json/schema/metadataIngestion/databaseServiceProfilerPipeline.json

```diff
@@ -35,10 +37,15 @@
       "default": true
     },
     "profileSample": {
-      "description": "Percentage of data used to execute the profiler for the whole workflow. This percentage will be applied to all tables in the workflow. Represented in the range (0, 100].",
+      "description": "Percentage of data we want to execute the profiler and tests on. Represented in the range (0, 100).",
       "type": "number",
-      "exclusiveMinimum": 0,
-      "maximum": 100,
+      "minimum": 0,
+      "default": null
+    },
+    "profileSampleUsingRows": {
```

Comment:
nit - maybe rename to `profileSampleRows`?

Reply:
also, 0 should be an exclusive minimum. If people don't want sample data they can disable the config

Reply:
makes sense..
making that change